src/main/java/picard/fingerprint/HaplotypeProbabilities.java

```diff
@@ -213,16 +220,16 @@ public double shiftedLogEvidenceProbabilityGivenOtherEvidence(final HaplotypePro
         if (!this.haplotypeBlock.equals(otherHp.getHaplotype())) {
             throw new IllegalArgumentException("Haplotypes are from different HaplotypeBlocks!");
         }
-        /** Get the posterior from the other otherHp. Use this posterior as the prior to calculate probability.
-         *
-         *   P(hap|x,y) = P(x|hap,y) P(hap|y) / P(x|y)
-         *              = P(x | hap) * P(hap | y) / P(x)
-         *                likelihood * other.posterior
-         *
-         *              = P(x|hap) P(y|hap) P(hap)/P(x)P(y)
-         *              = A P(x| hap) P(y| hap) P(hap)  # where A is an unknown scaling factor
+        /* Get the posterior from the other otherHp. Use this posterior as the prior to calculate probability.
+
+            P(hap|x,y) = P(x|hap,y) P(hap|y) / P(x|y)
+                       = P(x | hap) * P(hap | y) / P(x)
+                         likelihood * other.posterior
+
+                       = P(x|hap) P(y|hap) P(hap)/P(x)P(y)
+                       = A P(x| hap) P(y| hap) P(hap)  # where A is an unknown scaling factor
          */
-        return shiftedLogEvidenceProbabilityUsingGenotypeFrequencies(otherHp.getPosteriorProbabilities());
+        return shiftedLogEvidenceProbabilityUsingGenotypeFrequencies(otherHp.getPosteriorLikelihoods());
```

Comment:
For the reviewer, this is the bug fix....most of everything else is window dressing.